requirements.txt

```diff
@@ -37,7 +37,7 @@ six==1.16.0
 sniffio==1.3.0
 SQLAlchemy==1.4.39
 SQLAlchemy-Utils==0.41.1
-sqlglot-aidb==0.0.7
+sqlglot-aidb==0.0.9
```

Comment:
Oops why was this jumped by 2?

Reply:
0.0.8 is supporting user function, 0.0.9 is supporting where rewriter and join rewriter